.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie >= 9"]
```

Comment:
Not sure what browser support we should target. "last 2 versions" doesn't feel specific enough, and possibly we should hardcode Chrome 55 and Firefox 54, or something. 🤷‍♀️